stack.yaml

```diff
@@ -9,6 +9,7 @@ packages:
 - lib/test-utils
 - lib/jormungandr
 - lib/shelley
+- ../cardano-transactions
```

Comment:
this is intermediary, I just use this branch https://github.com/input-output-hk/cardano-transactions/pull/17. When this PR lands into master, we need to remove that change before merging this PR. but revert below change and bump commit